infra/main.bicepparam

```diff
@@ -13,6 +13,8 @@ param authType = readEnvironmentVariable('AZURE_AUTH_TYPE', 'keys')
 
 param hostingModel = readEnvironmentVariable('AZURE_APP_SERVICE_HOSTING_MODEL', 'code')
 
+param azureOpenAIModel = readEnvironmentVariable('AZURE_OPENAI_MODEL', 'gpt-4')
```

Comment:
This would mean that if we deploy from "Deploy to Azure" the default is 3.5; but deploying from `azd up` it would be 4.

Do you think it's best to leave as 3.5 until all regions support 4?

Reply:
Oops, my bad. I'll lower